GADOC.md

```diff
@@ -0,0 +1,757 @@
+
+## Overview
+InDesign APIs empower enterprises in automating tedious time-consuming design tasks via scalable cloud services, delivering personalization at scale, creative automation and content velocity. Our API is built on RESTful principles, employing standard HTTP response codes, methods, and authentication techniques to deliver responses in JSON format.
+
+InDesign APIs include three API capabilities:  
+* Rendition
+* Data merge 
+* Custom script
+
+This guide will assist you in:
+•	Setup and Creating a project within the Adobe Developer Console
```

Comment:
The list is not coming as bullets in Rich view, also sequence does not seem fine. 
Links & Working directory should come later